docs/references/javascript/organization-domain.mdx

```diff
@@ -0,0 +1,167 @@
+---
+title: OrganizationDomain
+description: The OrganizationDomain object is the model around an organization domain.
+---
+
+# `OrganizationDomain`
+
+The `OrganizationDomain` object is the model around an organization domain.
+
+## Attributes
+
+<Tables

```

Comment:
Because there are no complex components being used inside the table, let's opt for a markdown table so that it is more readable. `<Table />` components are reserved for more complex tables that involve component usage.

Instead, we can write:

| Name | Type | Description |
| --- | --- | --- |
| `id` | `string` | A unique identifier for this organization domain. |